### PR TITLE
feat: Profile link

### PR DESCRIPTION
**Pull request by dzbo**
### Ticket ID

[DEV-10538](https://app.clickup.com/t/2645698/DEV-10538)
[DEV-10559](https://app.clickup.com/t/2645698/DEV-10559)

### Description

- add profile link button

### CODE REVIEW

File: components/ProfileCardShareLink.vue [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  isCopied.value = true
+
+  setTimeout(() => {
+    hover.value = false
+    isCopied.value = false
+  }, 500)
+}
+
+const link = computed(() => {
+  return `https://m.profile.link/${props.resolvedName}`
```
Comment (franckwei) [inline components/ProfileCardShareLink.vue:26]: When I try to copy from the button, it copies with the profile address. But I thought the point was to copy the username + @ + first 4 digits of the profile address, so it's readable

Comment (dzbo) [reply components/ProfileCardShareLink.vue:26]: Apparently that is your resolved name. Can you send the link to the profile that you are testing?

Comment (franckwei) [reply components/ProfileCardShareLink.vue:26]: this is my profile address 0x404Ec4867ec5d4603e91C625014d05D95268cbc1

Comment (franckwei) [reply components/ProfileCardShareLink.vue:26]: image

Cool now I can see my username :) But the last digit seems cut

Comment (frozeman) [reply components/ProfileCardShareLink.vue:26]: Also the url is incorrect
it needs to be with out the `m.`

Comment (dzbo) [reply components/ProfileCardShareLink.vue]: Ok fixing

**Review by richtera — COMMENTED** [review]
Sorry I had some of my comments marked as pending.

File: components/ProfileCardShareLink.vue [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  }, 500)
+}
+
+const linkUrl = computed(() => {
+  if (props.profile?.profileLink?.isResolved) {
+    return props.profile?.profileLink?.link || ''
+  }
+
+  // if we can't resolve we just link to the wallet
+  return `${BASE_WALLET_URL}/${props.profile?.address}/?network=${currentNetwork.value.id}`
```
Comment (franckwei) [inline components/ProfileCardShareLink.vue:32]: not sure this works :/
image

Comment (franckwei) [reply components/ProfileCardShareLink.vue:32]: @richtera any idea?

Comment (franckwei) [reply components/ProfileCardShareLink.vue:32]: ah the last commit didn't pass the check, that's why

**Review by franckwei — APPROVED** [review]

**Review by richtera — APPROVED** [review]

**Comment by Hugoo** [comment]
Task linked: [DEV-10555 Profile appears as unknown on initial load](https://app.clickup.com/t/2645698/DEV-10555)

**Pull request merged**